### DISCUSSION TITLE
Hacks are being shut down. Thanks for everything.

### DISCUSSION THREAD

**Discussion by afkvido** [ProdigyPNP/ProdigyMathGameHacking#201] [Announcements]
**[Before you do anything, please join the temporary discord server, discord.gg/2qzTbdXSg9, though we'll be moving to other platforms soon.](https://discord.gg/2qzTbdXSg9)**

<br>

First we'd like to thank everyone for being with us. For the past 6 months, we've maintained the largest Prodigy hack, all while keepings things open source and ad-free.

Our goal, of bringing back Prodigy hacks, has somewhat been accomplished! While our hacks have fallen, [Prodigy X](https://github.com/ProdigyAPI/ProdigyX), a different cheat menu for Prodigy, has rose (It's partially closed source and does have ads and analytics. Always use [uBlock Origin](https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm)). Costs of keeping alive Prodigy Hack servers have also been up to $20 a month, in the past few months, and that's all from my, @afkvido's wallet. I make zero money from these hacks, and am going broke from them sometimes. Also, there have been trade negotiations of sorts with 

For these reasons, as of November 4, 2022, all official publications of our extension on webstores, and instances of P-NP servers, will all be closed down.

We will keep our source code on GitHub, where anyone can continue development by forking our repositories. We'll continue to review pull requests, but patched hacks most likely will remain patched. Our main issue is Prodigy made a little patch and I, being the only active person of ProdigyPNP, have no idea how to fix it.

If you would like to run your own instance of the hacks, check out [ProdigyPNP/P-NP](https://github.com/ProdigyPNP/P-NP), but again, there is still fixing to be done. [Our license allows anyone to run and develop our hacks, as long as no profit is made](https://github.com/ProdigyPNP/ProdigyMathGameHacking/blob/master/LICENSE.txt).

Thank you everyone for understanding, and getting us to insane milestones, such as 45k extension weekly users (before removal), 30k YouTube views (before our main tutorial was removed), and 7k Discord members (before the server was TOS'd).

If you're wondering where ProdigyPNP is headed next, which most of you likely aren't, and are just children thirsting for Prodigy hacks, sorry, anyways, I'm headed to developing actual software that's legal and doesn't break every five days, and **maybe, perhaps**, a [new Prodigy hack menu](https://github.com/ProdigyPXP). Check out @ProdigyPXP to see the software and for updates on the new hacks I'm making.

So long, everyone. Good luck, and GodSpeed.

**Comment by helloeveryonehoware** [comment]


**Reply by helloeveryonehoware** [reply]
Since I also wanna try learning typescript since that was a part of our lives and changed it for the better. Plus I've already forked it
I'm going to fork all of the ProdigyPNP stuff and mod it I'll still give credit I'm not a skidder

**Reply by helloeveryonehoware** [reply]
@afkvido should I fork infinite Zero

**Reply by afkvido** [reply]
> Should I try learning typescript so that I can mod the script and keep the legacy of ProdigyPNP

Sure. Good luck on that one though. Prodigy's Error 48 patch can only be fixed on P-NP itself, which is difficult enough by itself.

**Reply by afkvido** [reply]
> Since I also wanna try learning typescript since that was a part of our lives and changed it for the better. Plus I've already forked it I'm going to fork all of the ProdigyPNP stuff and mod it I'll still give credit I'm not a skidder

Thanks!

**Reply by afkvido** [reply]
> @afkvido should I fork infinite Zero

Sure, if you want. If you're gonna run it yourself, then probably yes. However, I've been thinking of better ways to patch Prodigy gamefiles, such as what ProdigyAPI did with GitHub actions.

**Reply by helloeveryonehoware** [reply]
afkvido I can teach you about lua code

**Reply by afkvido** [reply]
Sure. How are you gonna do that? Discord?

**Reply by helloeveryonehoware** [reply]
[Redacted by @ProdigyPNP due to malicious links]

**Reply by afkvido** [reply]
Can u join the server then send me a friend req?

**Reply by helloeveryonehoware** [reply]
Once I get home which is about 4 or 5 pm CDT

**Reply by afkvido** [reply]
alr

**Reply by helloeveryonehoware** [reply]
I sent a friend request on discord

**Reply by helloeveryonehoware** [reply]
> I sent a friend request on discord
@afkvido check your home and click pending to see if you have me on the pending list

**Reply by helloeveryonehoware** [reply]
[Redacted by @ProdigyPNP due to malicious links]

**Reply by afkvido** [reply]
Ah yes, the good old, "Disable your antivirus" trick. Please stop.

**Comment by UltraThyme42** [comment]
The same thing has occurred with Prodigy X.

**Comment by Chocodonutman** [comment]
Can we try and make a .exe filter for prodigy hack-blocker?

**Reply by afkvido** [reply]
Sir i don't think you know what you talking about

There's no need for exe's

It can be done with just a browser extension, modified browser settings, and a gamefile patcher (similar to P-NP, but pnp needs to be modified to work)

**Reply by afkvido** [reply]
But we could really use some help if you know any devs that would be great

**Comment by HIHIHIHIHIHIHIHI-ui** [comment]
I have a question what is the fill that souds more intrestin???

**Reply by HIHIHIHIHIHIHIHI-ui** [reply]
AND WHY IS THE HACKS GETTING SHUT DOWN!?
